src/core/Form/Checkbox/Checkbox.md

```diff
@@ -1,88 +1,212 @@
+The `<Checkbox>` component is used to mark a value as selected in a form.
+
+Examples:
+
+- [Basic use](/#/Components/Checkbox?id=basic-use)
+- [Hint text](/#/Components/Checkbox?id=hint-text)
+- [Error status](/#/Components/Checkbox?id=error-status)
+- [Default state](/#/Components/Checkbox?id=default-state)
+- [Controlled state](/#/Components/Checkbox?id=controlled-state)
+- [Disabled](/#/Components/Checkbox?id=disabled)
+- [Accessing value with ref](/#/Components/Checkbox?id=accessing-value-with-ref)
+- [Large checkbox](/#/Components/Checkbox?id=large-checkbox)
+- [Checkboxes in group](/#/Components/Checkbox?id=checkboxes-in-group)
+- [Group error status](/#/Components/Checkbox?id=group-error-status)
+- [Optional input](/#/Components/Checkbox?id=optional-input)
+- [Group with a tooltip](/#/Components/Checkbox?id=group-with-a-tooltip)
+
+<div style="margin-bottom: 5px">
+  [Props & methods (Checkbox)](/#/Components/Checkbox?id=props--methods)
+</div>
+<div style="margin-bottom: 40px">
+  [Props & methods (CheckboxGroup)](/#/Components/Checkbox?id=checkboxgroup)
+</div>
+
+### Basic use
+
+```js
+import { Checkbox } from 'suomifi-ui-components';
+
+<Checkbox>I accept the terms & conditions</Checkbox>;
+```
+
+### Hint text
+
+Use the `hintText` prop to provide additional information about the field
+
+```js
+import { Checkbox } from 'suomifi-ui-components';
+
+<Checkbox hintText="Choose this if you need personalized assistance">
+  I wish to be contacted regularly
+</Checkbox>;
+```
+
+### Error status
+
+Toggle the error status of the component using the `status` and `statusText` props.
```

Comment:
```suggestion
Toggle the error status of the component using the `status` and `statusText` props. The `statusText` is connected to the `<input>` using `aria-describedby` and read out to screen readers automatically using `aria-live="assertive"`.
```

Reply:
Should it be "read out to screen readers automatically **upon change**" just to be clear?

Reply:
I combined the two and it now reads 

The `statusText` is connected to the `<input>` using `aria-describedby` and read out to screen readers automatically upon change using `aria-live="assertive"`.